types/index.d.ts

```diff
@@ -69,7 +69,8 @@ export interface ClientMetadata {
   authorization_encrypted_response_alg?: string;
   authorization_encrypted_response_enc?: string;
   authorization_signed_response_alg?: string;
-
+  scopes_supported?: string[];
+  
```

Comment:
`scopes_supported` is not an [IANA registered client metadata property](https://www.iana.org/assignments/oauth-parameters/oauth-parameters.xhtml#client-metadata). Please remove this blob.

`scopes_supported` could potentially go to the `IssuerMetadata` interface, but as the library never reacts to its presence it's not included. I'm keen to only include properties that change the client/issuer behaviour. So it's either all or nothing and i think the inclusion of `[key: string]: unknown;` handles all possible metadata for anyone.